docker-compose.yml

```diff
@@ -767,6 +796,28 @@ services:
       - OPR_EXECUTOR_LOGGING_PROVIDER=$_APP_LOGGING_PROVIDER
       - OPR_EXECUTOR_LOGGING_CONFIG=$_APP_LOGGING_CONFIG
 
+  openruntimes-proxy:
```

Comment:
do we need the proxy for self-hosted ?

Reply:
Not for self-hosted, but keeping it in dev docker-compose ensures we always run with proxy (it tests proxy)